test/helpers/data.go

```diff
@@ -0,0 +1,39 @@
+package helpers
+
+import (
+	"math/rand"
+	"strings"
+	"sync"
+	"time"
+)
+
+const (
+	letterBytes   = "abcdefghijklmnopqrstuvwxyz"
+	randSuffixLen = 8
+	sep = "-"
+)
+
+var (
+	r        *rand.Rand
+	rndMutex *sync.Mutex
+	once     sync.Once
+)
+
+func initSeed() {
+	seed := time.Now().UTC().UnixNano()
+	r = rand.New(rand.NewSource(seed))
+	rndMutex = &sync.Mutex{}
+}
+
+func AppendRandomString(prefix string) string {
```

Comment:
the same function exists at https://github.com/knative/serving/blob/master/test/crd.go already. 
But it makes sense to move it out of serving to pkg. Will you pls remove it from serving?

Reply:
Sure, sorry I missed it

Reply:
Ah right, I was going to remove it right after the pr is in good shape :)

Reply:
Does it make sense to remove it from here as well? https://github.com/knative/build-pipeline/blob/master/test/randstring.go

Reply:
yes, all duplication

Reply:
can you pls open a git issue to track the next step?

Reply:
I've created one issue in this repository: https://github.com/knative/pkg/issues/274 and also two new ones on each repository to track the actual work:

https://github.com/knative/build-pipeline/issues/490
https://github.com/knative/serving/issues/3174